ml/db/init.sql

```diff
@@ -0,0 +1,170 @@
+-- INITALIZATION
+
+DROP DATABASE stratosphere;
+
+CREATE DATABASE stratosphere;
+ALTER DATABASE stratosphere OWNER TO postgres;
+
+\connect stratosphere
+
+ALTER USER postgres PASSWORD 'test1';
+
+-- RANKINGS
+
+-- Ranking relation
+CREATE TABLE ranking (
+	num 				int 		CHECK (num >= 0 AND num <= 36),
+	name 				text,
+	PRIMARY KEY (num));
+	
+-- Required ranking data
+INSERT INTO ranking (num, name) VALUES
+	(0,'Unranked'),
+	(1,'Bronze1'),(2,'Bronze2'),(3,'Bronze3'),
+	(4,'Silver1'),(5,'Silver2'),(6,'Silver3'),
+	(7,'Gold1'),(8,'Gold2'),(9,'Gold3'),
+	(10,'Platinum1'),(11,'Platinum2'),(12,'Platinum3'),
+	(13,'Diamond1'),(14,'Diamond2'),(15,'Diamond3'),
+	(16,'Champion1'),(17,'Champion2'),(18,'Champion3'),
+	(19,'GrandChamp1'),(20,'GrandChamp2'),(21,'GrandChamp3'),
+	(22,'SupersonicLegend'),
+	(23,'Prospect1'),(24,'Prospect2'),(25,'Prospect3'),(26,'ProspectElite'),
+	(27,'Challenger1'),(28,'Challenger2'),(29,'Challenger3'),(30,'ChallengerElite'),
+	(31,'RisingStar'),(32,'AllStar'),(33,'SuperStar'),
+	(34,'Champion'),(35,'GrandChampion'),(36,'Platinum');
+	
+-- PLATFORMS
+	
+-- Platform relation
+CREATE TABLE platform (
+	num					int			CHECK (num >= 0 AND num <= 3),
```

Comment:
Once again this can be [smallint](https://www.postgresql.org/docs/current/datatype-numeric.html)